packages/cli/test/config.spec.ts

```diff
@@ -21,17 +21,21 @@ describe('Stylable Cli Config', function () {
                 'package.json': `{"name": "test", "version": "0.0.0"}`,
                 'style.st.css': `.root{color:red}`,
                 'stylable.config.js': `
-                  exports.stcConfig = () => ({ options: { 
-                    outDir: './dist',
-                   } })
+                  exports.stcConfig = () => ({ 
+                      options: { 
+                            outDir: './dist',
+                            cjs: false
+                            esm: true,
+                        } 
+                    })
                 `,
             });
 
             runCliSync(['--rootDir', tempDir.path]);
 
             const dirContent = loadDirSync(tempDir.path);
             expect(Object.keys(dirContent)).to.eql([
-                'dist/style.st.css.js',
+                'dist/style.st.css.mjs',
```

Comment:
might want to check that there is no `cjs` file

Reply:
It checks for equality so these are the only files